tomviz/PythonGeneratedDatasetReaction.h

```diff
@@ -38,18 +38,15 @@ class PythonGeneratedDatasetReaction : public pqReaction
 
   void addDataset();
 
-  static vtkSmartPointer<vtkSMSourceProxy> getSourceProxy(const QString& label,
-                                                          const QString& script,
-                                                          const int shape[3]);
+  static DataSource* createGeneratedDataSource(
```

Comment:
As before - shorten to createDataSource - generated is already clear from the class name